docs/blog/posts/2024-02-28-reproducibility_tips.md

```diff
@@ -1,123 +1,109 @@
 ---
 date:
   created: 2024-02-28
+  updated: 2024-11-18
 categories:
     - Reproducibility
     - Research
+authors:
+    - kevin128
+subtitle: Reproducible Research Essentials
+
 ---
 # Reproducible Research Essentials
 
+This guide provides the foundational components needed to ensure reproducibility in your research. It focuses on:
```

Comment:
I wrote this guide to go with the hub how to. So maybe we [link it here](https://drive.google.com/file/d/1qfEL6qv1nyPAOAD8e0ziYLZuRwHOErlO/view?usp=sharing) too.

Reply:
Added a sentence and a link to the Research Hub Training & Workshop page.